packages/chronicle/src/themes/paper/Page.module.css

```
@@ -0,0 +1,154 @@
+.main {
+  --paper-navbar-height: 40px;
+  --paper-navbar-padding: var(--rs-space-3);
+  --paper-navbar-total: calc(var(--paper-navbar-height) + var(--paper-navbar-padding) * 2 + 1px);
+  flex: 1;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)